docs/topics/mta-rn-known-issues-7-0-2.adoc

```diff
@@ -11,12 +11,14 @@
 
 .Failure to connect to Jira server using basic authentication
 
-In {ProductShortName} 7.0.2, connecting to a Jira server using basic authentication, meaning _username_ and _password_, fails. This issue is scheduled to be resolved in {ProductShortName} 7.0.3. link:https://issues.redhat.com/browse/MTA-2427[(MTA-2427)]  
+In {ProductShortName} 7.0.2, connecting to a Jira server using basic authentication, meaning _username_ and _password_, fails. This issue is scheduled to be resolved in {ProductShortName} 7.0.3. link:https://issues.redhat.com/browse/MTA-2427[(MTA-2427)]
 
 .Unable to turn on Enable insecure communication switch
 
 In {ProductShortName} 7.0.2, it is not possible to turn on the `Enable insecure communication` switch when creating or editing a Jira instance. This issue is scheduled to be resolved in {ProductShortName} 7.0.3. link:https://issues.redhat.com/browse/MTA-2426[(MTA-2426)]
```

Comment:
This issue can be removed , it is fixed in 7.0.3

Reply:
This is 7.0.2 - known issues as I needed to include [MTA-2765](https://issues.redhat.com/browse/MTA-2765)